src/bindings/interface/analyservariable.i

```diff
@@ -0,0 +1,28 @@
+%module(package="libcellml") analyservariable
+
+#define LIBCELLML_EXPORT
```

Comment:
I'm never sure how messages should be constructed - infinitive or descriptive? ie: "Returns" or "Return"?

Reply:
Yeah, I have seen both being used and I think I may be using both too, although in different contexts. Maybe a candidate for issue #402?

Reply:
Sure.

Reply:
FTR, I have added a bullet point to issue #402.